content/collections/source-catalog/en/databricks.md

```diff
@@ -257,3 +257,15 @@ Depending on your company's network policy, you may need to add the following IP
    ```
     - **Root cause**: The delta log is truncated and expired, so the Amplitude service didn't find a delta log file to import.
     - **Solution**: Contact Amplitude support to skip expired data and continue the import. By default, retention is 30 days, make sure to keep the data for at least 7 days.
+
+6. ```
+   Fail worker job since databricks job run finished with state MAXIMUM_CONCURRENT_RUNS_REACHED.
+   ```
+   - **Root cause**: This means your Databricks job is configured to allow only one run at a time. If a new job is triggered while a previous run is still in progress, it will be skipped. This often happens when multiple Databricks import sources are set up to run at the same time and are using the same Databricks cluster.
+   - **Solution**: Update the Databricks job configuration to [increase the maximum concurrent runs](https://docs.databricks.com/aws/en/jobs/configure-job#configure-maximum-concurrent-runs).
+
+7. ```
+   [Databricks][JDBCDriver](700100) Connection timeout expired. Details: None.
+   ```
+   - **Root cause**: This means Amplitude was unable to establish a JDBC connection to the your databricks endpoint. This often happens because the cluster have been auto-terminated, not started yet or your databricks workspace may have hit limits (e.g., max concurrent SQL endpoints, cluster quota).
```

Comment:
```suggestion
   - **Root cause**: This means Amplitude was unable to establish a JDBC connection to the Databricks endpoint. This often happens because the cluster was automatically terminated, not started, or your Databricks workspace may have hit limits (for example, max concurrent SQL endpoints or cluster quota).
```